builder/files/config.toml

```diff
@@ -136,3 +136,7 @@ syncmode = "full"
 # [developer]
   # dev = false
   # period = 0
+
+# [parallelevm]
+  # enable = false
+  # specprocs = 8
```

Comment:
Let's use `procs` or `cores`, which is more straightforward.

Reply:
Done. Changed to `procs`.